Src/AutoFixtureUnitTest/EmailAddressLocalPartTest.cs

```diff
@@ -0,0 +1,169 @@
+            // Verify outcome
+            Assert.Equal(localPart, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void ToStringReturnsCorrectResult()
+        {
+            // Fixture setup
+            string expected = "good.localPart";
+            var sut = new EmailAddressLocalPart("good.localPart");
+            // Exercise system
+            var result = sut.ToString();
+            // Verify outcome
+            Assert.Equal(expected, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutIsEquatable()
+        {
+            // Fixture setup
+            // Exercise system
+            var sut = new EmailAddressLocalPart("good.localPart");
+            // Verify outcome
+            Assert.IsAssignableFrom<IEquatable<EmailAddressLocalPart>>(sut);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullObject()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            object other = null;
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullSut()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            UriScheme other = null;
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualAnonymousObject()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            var anonymousObject = new object();
+            // Exercise system
+            bool result = sut.Equals(anonymousObject);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualOtherObjectWhenSchemesDiffer()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            object other = new EmailAddressLocalPart("good2.localPart");
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualOtherSutWhenSchemesDiffer()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            var other = new EmailAddressLocalPart("good2.localPart");
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }      
```

Comment:
Could you pass this entire test suite by always returning `false` from `Equals`?